arbitrary-file-write/file-append/README.md

```diff
@@ -2,4 +2,49 @@
 * as root or user you can append data to a file and ...
 	* fully control content
 	* partially control content (eg. parts of a logfile)
-	* none (empty file)
\ No newline at end of file
+	* none (empty file)
+
+### sshd
+Looking at a mostly-default sshd config file most things are commented out so anything you append will take precedence if there is not something already before it (TODO: add sources for order of parsing).  
```

Comment:
this needs confirmation. What takes precedence? Can a second config overwrite the old one or not?